examples/make_workflow_diagram.ipynb

```diff
@@ -96,14 +96,14 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "D:/Work/Projects/MaMMoS/CHADA_MODA/mochada_kit/themes\n"
+      "/home/swapneel/Projects/Forks/mochada_kit/src/themes\n"
```

Comment:
I don't know if this matters here but the themes dir is actually outside the src dir (see other comment about the extra parent for the _THEMES_DIR in __init_.py).